client/core/core.go

```diff
@@ -1085,6 +1088,24 @@ func (c *Core) Trade(pw string, form *TradeForm) (*Order, error) {
 		return nil, fmt.Errorf("unknown DEX %s", form.DEX)
 	}
 
+	dc.suspensionMtx.RLock()
+	suspensionEpoch := dc.suspensionEpoch
+	dc.suspensionMtx.RUnlock()
+
+	// TODO: (Refinement) instead of outrightly avoiding placing orders when
+	// a trade suspension is received, we need to decide whether a trade
+	// can be taken based on the block times of the blockchains involved
+	// in the trade.
+	//
+	// timeRemaining := time.Duration(int64(suspensionEpoch) - time.Now().Unix())
+
+	// Avoid placing an order if the connection is scheduled for a
+	// trade suspension.
+	if suspensionEpoch != 0 {
+		return nil, fmt.Errorf("order placement suspended due to impending"+
+			" trade suspension for dex %s", dc.acct.url)
```

Comment:
It's worth saying when this suspension is going to happen. (suspension epoch * epoch duration (ms))  We'd need to get this from the market map, but as @buck54321 pointed out, there will need to be a suspension epoch for each market.